lib/cartopy/mpl/gridliner.py

```diff
@@ -335,9 +336,10 @@ def _draw_gridliner(self, nx=None, ny=None, background_patch=None):
         if self.xlines:
             lines = []
             for x in x_gridline_points:
-                l = list(zip(np.zeros(n_steps) + x,
-                         np.linspace(min(y_ticks), max(y_ticks), n_steps)))
-                lines.append(l)
+                ticks = list(zip(
```

Comment:
I could've sworn I'd written some optimization for this unfortunate type conversion; maybe I never made a PR out of it, but I can see if I can dig it up.

Reply:
I don't see a need to connect this with the issue at hand  ?
We rather want to get this onto master so we can get on right now, since stickler is now complaining about various open PRs.

Reply:
No, just a reminder to myself.